demisto_sdk/commands/init/initiator.py

```diff
@@ -556,13 +565,15 @@ def get_remote_templates(self, files_list, dir):
         Returns:
             bool. True if the files were downloaded and saved successfully, False otherwise.
         """
-        if self.template in [self.HELLO_WORLD_INTEGRATION] + self.DEFAULT_TEMPLATES:
+        # create test_data dir
+        if self.template in [self.HELLO_WORLD_INTEGRATION] + self.DEFAULT_TEMPLATES\
+                + [self.HELLO_WORLD_FEED_INTEGRATION]:
             os.mkdir(os.path.join(self.full_output_path, self.TEST_DATA_DIR))
 
         if self.template in self.DEFAULT_TEMPLATES:
             pack_name = self.DEFAULT_TEMPLATE_PACK_NAME
 
-        elif self.template in self.HELLO_WORLD_TEMPLATES:
+        elif self.template in self.HELLO_WORLD_TEMPLATES + [self.HELLO_WORLD_FEED_INTEGRATION]:
```

Comment:
why `HELLO_WORLD_FEED_INTEGRATION` is not included in the `HELLO_WORLD_TEMPLATES` list?